Makefile.core.mk

```diff
@@ -73,13 +73,20 @@ endif
 # Which branch of the Istio source code do we fetch stuff from
 SOURCE_BRANCH_NAME ?= master
 
-gen:
+site:
 	@scripts/gen_site.sh
 
-build: gen
+snips:
+	@scripts/gen_snips.sh
+
+gen: snips
```

Comment:
I am not familiar with this, does this not need `site` as well?

Reply:
I don't think so. `make site` generates and serves the site. That's why I'm renaming it from `gen` to `site` and just using `make gen` to generate the artifacts, so `gen-check` does the right thing.